.travis.yml

```diff
@@ -1,35 +1,44 @@
-dist: trusty  # Needed for C++11 support.
 language: cpp
 sudo: required
 
-compiler:
-    - clang
-    - gcc
-
 before_install:
     - ./admin/travis-install-deps.sh
 
-env:
-    - ARCH=amd64 DO=apidocs
-    - ARCH=amd64 DO=style
-    - ARCH=amd64 DO=distcheck AS_ROOT=no
-    - ARCH=amd64 DO=distcheck AS_ROOT=yes UNPRIVILEGED_USER=no
-    - ARCH=amd64 DO=distcheck AS_ROOT=yes UNPRIVILEGED_USER=yes
-    - ARCH=i386 DO=distcheck AS_ROOT=no
-
 matrix:
-    exclude:
-        # Treat clang as the main compiler.  For gcc, just run the AS_ROOT=no
-        # env above to see if we can actually build, but do not really worry
-        # about the run-time aspects of the tests.
-        - compiler: gcc
+    include:
+        - os: linux
+          dist: xenial
+          compiler: clang
+          env: ARCH=amd64 DO=distcheck AS_ROOT=no
+        - os: linux
+          dist: xenial
+          compiler: gcc
+          env: ARCH=amd64 DO=distcheck AS_ROOT=no
+        - os: linux
+          dist: xenial
+          compiler: clang
           env: ARCH=amd64 DO=apidocs
-        - compiler: gcc
+        - os: linux
+          dist: xenial
+          compiler: clang
           env: ARCH=amd64 DO=style
-        - compiler: gcc
+        - os: linux
+          dist: xenial
+          compiler: clang
           env: ARCH=amd64 DO=distcheck AS_ROOT=yes UNPRIVILEGED_USER=no
-        - compiler: gcc
+        - os: linux
+          dist: xenial
+          compiler: clang
           env: ARCH=amd64 DO=distcheck AS_ROOT=yes UNPRIVILEGED_USER=yes
+        # XXX: reenable i386
```

Comment:
Please use the same syntax elsewhere (`grep -r TODO`):

```
# TODO(username): Reenable i386 once blah blah blah.
```

(The username should be someone that knows the answer and is not an assignment of work. But you can put mine if you don't feel like encoding yours.)

In other words... explain why this is broken now and what should happen. As things are right now, I cannot tell why i386 had to be disabled nor what it'd take to fix it.

Reply:
Done :)!